### PR TITLE
feat(admin-setting): clear Give Cache from within Advanced tab #2719

### LINKED ISSUE #2719: feat(admin-setting): clear Give Cache from within Advanced tab
## Issue Overview

Add Clear Give_Cache Button in "Donations > Settings > Advanced" 

We have a Give_Cache now, and it is relatively smart about clearing at the right times; but for peace of mind and troubleshooting purposes, we should add the ability to force clear it.

## Tasks
- [x] Add setting to delete cache via ajax
- [x] Test it with some caching system like Redis and Memcached.

### PR DESCRIPTION
**Pull request by Sidsector9**
Closes #2719 

## Description
Added a button to clear Give's cache on the **Advanced** tab of the **Give Settings** page.

## How Has This Been Tested?
Tested and failed, needs senior developer intervention.

## Screenshots (jpeg or gifs if applicable):
clearcache

## Types of changes
**New feature:** Users will now be able to clear cache with a dedicated button.

## Checklist:
- [ ] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: assets/src/js/admin/admin-settings.js [hunk 1]
```diff
@@ -130,4 +130,24 @@ jQuery(document).ready(function ($) {
...
+		$.ajax({
+			url: ajaxurl,
+			type: 'GET',
+			data: {
+				action: 'give_cache_flush'
+			}
+		})
+		.done( function( response ) {
+			if ( response.success ) {
+				alert( 'Cache flushed successfully.' );
```
Comment (raftaar1191) [inline assets/src/js/admin/admin-settings.js]: Add string translation ( Text Domain )

File: assets/src/js/admin/admin-settings.js [hunk 2]
```diff
@@ -130,4 +130,24 @@ jQuery(document).ready(function ($) {
...
+			} else {
+				alert( 'An error occured while flushing the cache.' );
```
Comment (raftaar1191) [inline assets/src/js/admin/admin-settings.js]: Add string translation ( Text Domain )

**Comment by ravinderk** [comment]
@DevinWalker this PR ready to merge. I need help with the following thing:
- [ ] Review text:
   - [ ] Notice: https://github.com/WordImpress/Give/pull/2948/files#diff-e9c95b3cdaf1809941dce657070cec83R1132
   - [ ] Notice: https://github.com/WordImpress/Give/pull/2948/files#diff-e9c95b3cdaf1809941dce657070cec83R1136
    - [ ] Field descriotion and name: https://github.com/WordImpress/Give/pull/2948/files#diff-46921ffadf0827a34b0a0cdecc1b3989R114

**Comment by Sidsector9** [comment]
This is the revised output after Ravinder's suggestion - 

clearcache

**Pull request merged**